commit 86c7acddd405b256597031e50e1626e66e452dc5
Author: Martin Cermak <mcermak@redhat.com>
Date:   Tue Feb 24 17:48:46 2015 +0100

    For rhel6 kernels, make the execveat syscall optional in [nd_]syscalls-detailed.stp.

diff --git a/testsuite/buildok/nd_syscalls-detailed.stp b/testsuite/buildok/nd_syscalls-detailed.stp
index e759d1d..a7c57b0 100755
--- a/testsuite/buildok/nd_syscalls-detailed.stp
+++ b/testsuite/buildok/nd_syscalls-detailed.stp
@@ -342,13 +342,13 @@ probe nd_syscall.execve.return, nd_syscall.compat_execve.return ?
 	printf("%s, %s\n", name, retstr)
 }
 
-probe nd_syscall.execveat, nd_syscall.compat_execveat ?
+probe nd_syscall.execveat ?, nd_syscall.compat_execveat ?
 {
         printf("%s, %s\n", name, argstr)
         printf("%d, %s, %s, %d(%s), %s\n", fd, fd_str, filename, flags, flags_str, args)
 }
 
-probe nd_syscall.execveat.return, nd_syscall.compat_execveat.return ?
+probe nd_syscall.execveat.return ?, nd_syscall.compat_execveat.return ?
 {
         printf("%s, %s\n", name, retstr)
 }
diff --git a/testsuite/buildok/syscalls-detailed.stp b/testsuite/buildok/syscalls-detailed.stp
index f3f3ddc..230dd9a 100755
--- a/testsuite/buildok/syscalls-detailed.stp
+++ b/testsuite/buildok/syscalls-detailed.stp
@@ -342,13 +342,13 @@ probe syscall.execve.return, syscall.compat_execve.return ?
 	printf("%s, %s\n", name, retstr)
 }
 
-probe syscall.execveat, syscall.compat_execveat ?
+probe syscall.execveat ?, syscall.compat_execveat ?
 {
         printf("%s, %s\n", name, argstr)
 	 printf("%d, %s, %s, %d(%s), %s\n", fd, fd_str, filename, flags, flags_str, args)
 }
 
-probe syscall.execveat.return, syscall.compat_execveat.return ?
+probe syscall.execveat.return ?, syscall.compat_execveat.return ?
 {
         printf("%s, %s\n", name, retstr)
 }
